lib/routes/nextjs/blog.ts

```diff
@@ -0,0 +1,57 @@
+import type { DataItem, Route } from '@/types';
+import { parseDate } from '@/utils/parse-date';
+import { load } from 'cheerio';
+import ofetch from '@/utils/ofetch';
+
+const handler: Route['handler'] = async () => {
+    const data = await ofetch('https://nextjs.org/blog');
+
+    const $ = load(data);
+
+    const item = await Promise.all(
+        $('article')
+            .toArray()
+            .slice(0, 20)
+            .map<Promise<DataItem>>(async (item) => {
+                const $ = load(item);
+
+                const h = $('a[href^="/blog"]');
+                const title = h.text().trim();
+                const link = `https://nextjs.org${h.attr('href')}`;
+                const date = $('p').first().text(); // not reliable, but works for now
+
+                const data = await ofetch(link);
+
+                return {
+                    title,
+                    link,
+                    description: load(data)('div.prose').html() ?? '',
+                    pubDate: parseDate(date.replace(/st|nd|rd|th/, '')),
+                };
```

Comment:
Please [cache](https://docs.rsshub.app/joinus/advanced/use-cache) this request, since the content is a less frequently updated content.